demo/js/components/canvas-demo.js

```diff
@@ -0,0 +1,263 @@
+      { year: 1975, value: 916395000 },
+      { year: 1976, value: 930685000 },
+      { year: 1977, value: 943455000 },
+      { year: 1978, value: 956165000 },
+      { year: 1979, value: 969005000 },
+      { year: 1980, value: 981235000 },
+      { year: 1981, value: 993885000 },
+      { year: 1982, value: 1008630000 },
+      { year: 1983, value: 1023310000 },
+      { year: 1984, value: 1036825000 },
+      { year: 1985, value: 1051040000 },
+      { year: 1986, value: 1066790000 },
+      { year: 1987, value: 1084035000 },
+      { year: 1988, value: 1101630000 },
+      { year: 1989, value: 1118650000 },
+      { year: 1990, value: 1135185000 },
+      { year: 1991, value: 1150780000 },
+      { year: 1992, value: 1164970000 },
+      { year: 1993, value: 1178440000 },
+      { year: 1994, value: 1191835000 },
+      { year: 1995, value: 1204855000 },
+      { year: 1996, value: 1217550000 },
+      { year: 1997, value: 1230075000 },
+      { year: 1998, value: 1241935000 },
+      { year: 1999, value: 1252735000 },
+      { year: 2000, value: 1262645000 },
+      { year: 2001, value: 1271850000 },
+      { year: 2002, value: 1280400000 },
+      { year: 2003, value: 1288400000 },
+      { year: 2004, value: 1296075000 },
+      { year: 2005, value: 1303720000 },
+      { year: 2006, value: 1311020000 },
+      { year: 2007, value: 1317885000 },
+      { year: 2008, value: 1324655000 },
+      { year: 2009, value: 1331260000 },
+      { year: 2010, value: 1337705000 },
+      { year: 2011, value: 1344130000 },
+      { year: 2012, value: 1350695000 },
+      { year: 2013, value: 1357380000 },
+      { year: 2014, value: 1364270000 },
+      { year: 2015, value: 1371220000 },
+      { year: 2016, value: 1378665000 },
+      { year: 2017, value: 1386395000 },
+      { year: 2018, value: 1392730000 },
+      { year: 2019, value: 1397715000 },
+      { year: 2020, value: null }
+    ]
+  }
+];
+
+const containerStyle = {
+  display: "flex",
+  flexDirection: "row",
+  flexWrap: "wrap",
+  alignItems: "center",
+  justifyContent: "center"
+};
+
+const parentStyle = {
+  border: "1px solid #ccc",
+  margin: "2%",
+  maxWidth: "40%"
+};
+
+const formatPopulation = (value) => {
+  const ONE_BILLION = 1000000000;
+  const ONE_MILLION = 1000000;
+  if (value >= ONE_BILLION) {
+    return `${value / ONE_BILLION}B`;
+  } else if (value >= ONE_MILLION) {
+    return `${value / ONE_MILLION}M`;
+  }
+  return value;
+};
+
+const CanvasDemo = () => {
+  return (
+    <div className="demo">
+      <div style={containerStyle}>
+        <VictoryChart style={{ parent: parentStyle }}>
+          {populationData.map(({ country, values }) => {
+            const data = values.map(({ year, value }) => ({
+              x: year,
+              y: value
+            }));
+            return (
+              <VictoryLine
+                key={country}
+                data={data}
+                groupComponent={<CanvasContainer />}
```

Comment:
I really like providing the canvas as the `groupComponent` here so that we leave open the possibility of using the interactive containers with canvas. This might be a motivation for coming up with a different name than `CanvasContainer` though, to avoid confusion about how it should be used.

Reply:
Yes, that's a good thought!